legion_test/bin/dex_client

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+#
+#    Copyright 2018 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+"""
+Tool authenticates in Dex and returns Jenkins token and cookies
+"""
+import argparse
+from legion_test.robot import dex_client
+from legion_test import profiler_loader
+import os
+
+def work():
+    profiler_loader.get_variables(os.getenv('PATH_TO_PROFILES_DIR'))
+    if dex_client.get_jenkins_credentials():
+        username, password = dex_client.get_jenkins_credentials()
+        cookies = ';'.join(['{}={}'.format(k, v) for (k, v) in dex_client.get_session_cookies().items()])
+        print('{}\n{}\n{}\n'.format(username, password, cookies))
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser('Dex client')
```

Comment:
@aliaksandr-d maybe we should rename this tool to make it clear that it generates only Jenkins credentials (like jenkins-dex-client)?